src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -33,6 +33,10 @@
  */
 class TranslationExtension extends AbstractExtension
 {
+    use TranslatorTrait {
+        trans as private doTrans;
```

Comment:
I don't understand this: getTranslator already provide an identity translator as a fallback. I removed using the trait to allow instantiating this class without the translation-contracts. This revert this.